RELEASE-NOTES.md

```diff
@@ -12,6 +12,8 @@
 - Sampling from variational approximation now allows for alternative trace backends [#3550].
 - Infix `@` operator now works with random variables and deterministics [#3619](https://github.com/pymc-devs/pymc3/pull/3619).
 - [ArviZ](https://arviz-devs.github.io/arviz/) is now a requirement, and handles plotting, diagnostics, and statistical checks.
+- Can use GaussianRandomWalk in sample_prior_predictive [#3682](https://github.com/pymc-devs/pymc3/pull/3682)
```

Comment:
Not only that, it can now be used in `sample_posterior_predictive` even if it is an `ObservedRV`

Reply:
good point!